test/integration/index.js

```diff
@@ -4,18 +4,23 @@ const knex = require('../../knex');
 const logger = require('./logger');
 const config = require('../knexfile');
 const fs = require('fs');
+const path = require('path');
 
 const Bluebird = require('bluebird');
 
 Object.keys(config).forEach((dialectName) => {
   return require('./suite')(logger(knex(config[dialectName])));
 });
 
-after(function(done) {
+before(function(done) {
   if (config.sqlite3 && config.sqlite3.connection.filename !== ':memory:') {
-    fs.unlink(config.sqlite3.connection.filename, function() {
-      done();
-    });
+    fs.copyFile(
```

Comment:
Is this necessary for all tests in this suite?

Reply:
We can reuse this copied database for other suites too, they created the test.sqlite3 anyway if it does not exist.

Reply:
But it's a very specific DB for very specific case, why would anyone want to reuse it for unrelated test? And ideally tests should be as isolated as possible, to avoid breaking lost of tests when making unrelated change to test resources.

Reply:
Yes, I agree absolutely, tests should be isolated from each other. I changed the code, see my other comment.